dashboard/src/components/DatasetOverview.tsx

```diff
@@ -1,21 +1,45 @@
 import { TbDatabasePlus } from "solid-icons/tb";
-import { Show, For, Setter, Accessor, createEffect } from "solid-js";
-import { DatasetAndUsage } from "../types/apiTypes";
+import {
+  Show,
+  For,
+  Setter,
+  Accessor,
+  createSignal,
+  createEffect,
+} from "solid-js";
+import { Organization } from "../types/apiTypes";
 import { useNavigate } from "@solidjs/router";
 import { FiTrash } from "solid-icons/fi";
 import { FaSolidGear } from "solid-icons/fa";
+import { useDatasetPages } from "../hooks/useDatasetPages";
+import { AiFillCaretLeft, AiFillCaretRight } from "solid-icons/ai";
+import { formatDate } from "../formatters";
 
 export interface DatasetOverviewProps {
   setOpenNewDatasetModal: Setter<boolean>;
-  datasetAndUsages: Accessor<DatasetAndUsage[]>;
-  setDatasetsAndUsages: Setter<DatasetAndUsage[]>;
+  selectedOrganization: Accessor<Organization>;
 }
 
 export const DatasetOverview = (props: DatasetOverviewProps) => {
   const navigate = useNavigate();
+  const [page, setPage] = createSignal(0);
+  const [datasetSearchQuery, setDatasetSearchQuery] = createSignal("");
+
+  createEffect(() => {
+    props.selectedOrganization();
+    setPage(0);
+  });
+
+  const { datasets, maxPageDiscovered, removeDataset, hasLoaded } =
```

Comment:
Generally destructuring in Solid should be avoided. I think it's fine here though.